tests/cases/fourslash/completionListForStringUnion.ts

```diff
@@ -0,0 +1,11 @@
+/// <reference path='fourslash.ts' />
+
+//// type A = 'fooooo' | 'barrrrr';
+//// type B<T extends A> = {};
+//// type C = B<'fooooo' | '/**/'>
+
+
+goTo.marker();
+// verify.completionListIsEmpty();
```

Comment:
whats with the commented out code.? You want to be able to show completion even when b is not inserted in the string.

Reply:
Sorry, that is a mistake.